src/fn/fn-table.scss

```diff
@@ -0,0 +1,175 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-table;
+$fn-table-item-margin: 0.375rem;
+$fn-table-item-focus-margin: 0.125rem;
+$fn-table-item-box-shadow: 0 0 0 $fn-table-item-focus-margin;
+
+.#{$block} {
+  @include fn-reset();
+
+  width: 100%;
+  max-width: 100%;
+  border-collapse: collapse;
+
+  &__row {
+    @include fn-reset();
+
+    @include fn-selected() {
+      .#{$block}__cell {
+        background: $fn-color-blue-2;
+        box-shadow: inset 0 -0.063rem 0 #0070f1;
+      }
+    }
+  }
+
+  .#{$block}__text {
+    @include fn-reset();
+    @include fn-ellipsis();
+
+    word-break: break-word;
+    white-space: normal;
+
+    &--no-wrap {
+      white-space: nowrap;
+    }
+  }
+
+  &__cell,
+  &__group-header,
+  &__header {
+    @include fn-reset();
+
+    box-shadow: inset 0 -0.063rem 0 #e4e5e7;
+  }
+
+  &__cell,
+  &__group-header {
+    background: $fn-color-white;
+    height: 2.75rem;
+  }
+
+  &__group-header,
+  &__header {
+    text-transform: uppercase;
+    text-align: start;
+    font-size: 0.75rem;
+    line-height: 1.125rem;
+    font-weight: bold;
+
+    .#{$block}__text--end {
+      @include fn-set-margin-left(auto);
+    }
+
+    .#{$block}__header-checkbox {
+      @include fn-set-margin-left(-0.25rem);
+    }
+  }
+
+  &__group-header {
+    padding: 1.25rem 0.75rem 0.375rem;
+  }
+
+  &__header {
+    background: $fn-color-grey-2;
+    height: 2rem;
+
+    @include fn-set-padding-left(1rem);
+
+    &:not(:first-of-type) {
+      @include fn-set-padding-left(0.75rem);
+    }
+  }
+
+  &__cell {
+    &:not(.fn-table__cell--no-hover) {
+      @include fn-hover() {
+        background: $fn-color-grey-1;
+      }
+    }
+
+    @include fn-focus-visible() {
+      outline: none;
+      border-radius: $fn-table-item-focus-margin;
+      position: relative;
+    }
+
+    @include fn-not-focus-visible() {
+      @include fn-hover() {
+        background: $fn-color-grey-1;
+      }
+    }
+
+    @include fn-selected() {
+      background: $fn-color-blue-2;
+      box-shadow: inset 0 -0.063rem 0 #0070f1;
+    }
+
+    padding: $fn-table-item-margin 0.75rem;
+
+    .#{$block}__icon {
+      @include fn-set-margins-x(0.25rem, 0.625rem);
+
+      color: $fn-color-grey-7;
+    }
+
+    @include fn-disabled() {
+      pointer-events: none;
+      opacity: $fn-disabled-opacity;
+    }
+
+    .#{$block}__avatar {
+      @include fn-set-margin-right($fn-table-item-margin);
+    }
```

Comment:
you can avoid this additional css class by relying of first-child.
With byline list you always have `.fn-table__start-container` element. When you have an avatar in front of it, this element is never the first child, so in this case you can add margin-left. Something like:
```
.fn-table__start-container:not(:first-child) {
    @include fn-set-margin-left($fn-table-item-margin);
}
```

This will add the spacing only when the element has a sibling before it and the good part is that it will be valid for any element not just avatar